eclipse-scout-core/src/tile/fields/tablefield/TileTableField.less

```diff
@@ -53,6 +53,27 @@
       & > .table-header-item {
         color: @tile-table-default-color;
 
+        &.sorted,
+        &.grouped,
+        &.filtered {
+          padding-right: @table-header-item-padding-right;
+
+          &.halign-right,
+          &.halign-center {
+            padding-right: @table-cell-padding-left;
+          }
+        }
+
+        &.sorted.grouped,
+        &.sorted.filtered {
```

Comment:
Please add the rules to the ones above, it should not be necessary to duplicate them. Actually, they should not be necessary at all because the rules for the tile table field are already more specific than the ones from the TableHeader.less.